cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -25,108 +26,329 @@
     )
 ]
 
-vms = []
 
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal],
+                            ['nfs', 'iscsi', rhel7_minimal], ['iscsi', 'iscsi', rhel7_minimal],
+                            ['iscsi', 'nfs', rhel7_minimal], ['iscsi', 'local', rhel7_minimal]],
+                        indirect=True)
+def test_single_datastore_single_vm_migration(request, appliance, v2v_providers, host_creds,
+                                            conversion_tags,
+                                            form_data_vm_obj_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore[0])
+    request.addfinalizer(lambda: infrastructure_mapping_collection.delete(mapping))
+
+    migration_plan_collection = appliance.collections.v2v_plans
+
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=form_data_vm_obj_single_datastore[1], start_migration=True)
+
+    # explicit wait for spinner of in-progress status card
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    def _get_plan_status():
+        is_plan_visible = view.progress_card.is_plan_visible(migration_plan.name)
+        if is_plan_visible:
+            # log current status
+            # uncomment following logs if github.com/ManageIQ/miq_v2v_ui_plugin/issues/415 fixed
```

Comment:
This issue has been closed/resolved, please document any remaining blockers to this.

Reply:
@mshriver I am going to add this new xpaths in next PR related to new UI changes.